libcudacxx/include/cuda/std/__cccl/compiler.h

```diff
@@ -19,7 +21,7 @@
       "Support for the Intel C++ Compiler Classic is deprecated and will eventually be removed. Define CCCL_SUPPRESS_ICC_DEPRECATION_WARNING to suppress this warning"
 #  endif // CCCL_SUPPRESS_ICC_DEPRECATION_WARNING
 #elif defined(__NVCOMPILER)
-#  define _CCCL_COMPILER_NVHPC
+#  define _CCCL_COMPILER_NVHPC _CCCL_COMPILER_MAKE_VERSION(__NVCOMPILER_MAJOR__, __NVCOMPILER_MINOR__)
 #  define _CCCL_COMPILER_NVHPC_VERSION \
     (__NVCOMPILER_MAJOR__ * 10000 + __NVCOMPILER_MINOR__ * 100 + __NVCOMPILER_PATCHLEVEL__)
```

Comment:
We should drop that in favor of just `_CCCL_COMPILER_NVHPC`
```suggestion
```

Reply:
I wanted to keep all compiler version macros unchanged for now as Thrust compiler marcos depend on them and remove them in separate PR. But the fact is that nvhpc version macros is unused, so there is no problem removing it now :)